code/modules/admin/view_variables/reference_tracking.dm

```diff
@@ -0,0 +1,161 @@
+#ifdef REFERENCE_TRACKING
```

Comment:
did you give ref tracking a test ? i don't remember for sure if it was fully added in on SSgarbage during backport. either way it's good to have in the future

Reply:
it didn't - now it does!